botocore/parsers.py

```diff
@@ -210,6 +210,7 @@ class ResponseParser:
 
     DEFAULT_ENCODING = 'utf-8'
     EVENT_STREAM_PARSER_CLS = None
+    KNOWN_LOCATIONS = ('header', 'headers', 'status_code')
```

Comment:
I mentioned this in the last roud of reviews and I still think it's a good idea to leave a comment here to provide more context:

```suggestion
    # This is a list of known values for the "location" key in the
    # serialization dict. The location key tells us where in the response
    # to parse the value. Locations not in this list will be ignored.
    KNOWN_LOCATIONS = ('header', 'headers', 'status_code')
```

Reply:
There is a comment about it inside the if statement where `KNOWN_LOCATIONS` is used, so I didn't think commenting here was necessary.

```
Locations not in this list will be ignored.
```

Plus, aren't we ignoring the locations in this list?

Reply:
> Plus, aren't we ignoring the locations in this list?

At a high-level, no, we aren't ignoring members with locations that are know. We will parse them from the location specified (e.g. We will parse from the header or status code). However, when checking for which members to parse from the body, we do ignore them since they were already parsed somewhere else.

I updated the suggested comment above with hopefully more clear wording.